src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -2224,32 +2226,41 @@ private void OnRigidityChanged()
 
         gui.UpdateSpeed(CalculateSpeed());
         gui.UpdateHitpoints(CalculateHitpoints());
+
+        DirtyMutationPointsCache();
     }
 
     /// <summary>
     ///   Perform all actions through this to make undo and redo work
     /// </summary>
-    private void EnqueueAction(MicrobeEditorAction action)
+    /// <returns>True when the action was successful</returns>
+    private bool EnqueueAction(MicrobeEditorAction action)
     {
+        var historyClone = (EditorActionHistory)history.Clone();
+        historyClone.AddAction(action);
```

Comment:
As the new microbe action holds the old organelle list, I think this clone is just as bad as the clone in the previous attempt at dynamic MP was (that cloned a species).

Reply:
![image](https://user-images.githubusercontent.com/60734649/127279500-2bf649e9-dcd0-40cc-bb81-ab333884cef9.png)
Better that the last one. This can be easily avoided.